Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -78,13 +74,21 @@ inline CGPoint __CreateCGPointWithTransform(CGFloat x, CGFloat y, const CGAffine
         return _impl.startingPoint;
     }
 
+    void SetIsFigureClosed(bool closed) {
+        _impl.isFigureClosed = closed;
+    }
+
+    bool GetIsFigureClosed() {
```

Comment:
Would rather not overload the variable name with a capitalized version of a getter. The naming convention for functions says to use isFigureClosed so I might as well change the entire variable name at this point. However, we're just dancing around semantics at this point.

---
In reply to: [85826920](https://github.com/Microsoft/WinObjC/pull/1255#discussion_r85826920) [](ancestors = 85826920)